README.md

```diff
@@ -26,6 +26,13 @@ Install-Module DockerCompletion -Scope CurrentUser
 # Import
 Import-Module DockerCompletion
 ```
+
+### Auto Import
+You can add import to your profile to avoid calling it each time you open shell:
```

Comment:
A link for the official document explaining PowerShell profile should be added.

Adding `Import-Module` to profile is a well-known technique. The people who does not know it is probably a beginner, not knowing what profile is.